src/library/Ux4iot.ts

```diff
@@ -108,13 +108,12 @@ export class Ux4iot {
 		this.axiosInstance = axios.create({
 			baseURL: ux4iotURL,
 		});
-		this.socketState = 'INIT';
 		this.devMode = devMode;
 	}
 
 	private log(...args: any[]) {
 		if (this.devMode) {
-			console.log('Ux4iot:', ...args);
+			console.warn('Ux4iot:', ...args);
```

Comment:
Please always use lower-case, i.e. ux4iot instead of Ux4iot
```suggestion
			console.warn('ux4iot:', ...args);
```